.gitignore

```diff
@@ -3,3 +3,4 @@ _site/
 .jekyll-cache/
 .jekyll-metadata
 Gemfile.lock
+assets/courses/.DS_Store
```

Comment:
🤣 lmfaooo, yeah I think this will potentially prevent some annoyance for some people, let's go with it :D

Reply:
:D !!